javascript/organizing_your_javascript_code/webpack.md

```diff
@@ -408,6 +408,12 @@ Secondly, by default, `webpack-dev-server` will only auto-restart when it detect
 
 Once set up, `npx webpack serve` will host our web page on `http://localhost:8080/`, which we can open in our browser and start working!
 
+<div class="lesson-note">
```

Comment:
```suggestion
<div class="lesson-note" markdown="1">
```

Reply:
My bad - I used the "commit suggestion" feature and didn't realize it wasn't one, but two separate suggestoins.

Should be good to go now